storage/remote/queue_manager.go

```diff
@@ -303,7 +306,9 @@ func NewQueueManager(
 		samplesOut:         newEWMARate(ewmaWeight, shardUpdateDuration),
 		samplesOutDuration: newEWMARate(ewmaWeight, shardUpdateDuration),
 
-		metrics: metrics,
+		metrics:              metrics,
+		interner:             newPool(),
```

Comment:
Creating a new interner for each queue_manager will significantly increase the amount of memory used by remote write when more than one destination is configured. If we want to stop using a global interner, it should be created in https://github.com/prometheus/prometheus/blob/master/storage/remote/write.go#L64 most likely.